react-bootstrap/react-bootstrap.d.ts

```diff
@@ -943,8 +943,8 @@ declare module "react-bootstrap" {
         Addon: typeof InputGroupAddon;
         Button: typeof InputGroupButton;
     }
-    type InputGroup = React.ClassicComponent<InputGroupProps, {}>;
-    var InputGroup: React.ClassicComponentClass<InputGroupProps>;
+    type InputGroup = React.Component<InputGroupProps, {}>;
+    var InputGroup: InputGroupClass;
```

Comment:
That's not necessarily the right pattern to follow. `ClassicComponentClass` should only be used for components created with `React.createClass`—[InputGroup does not](https://github.com/react-bootstrap/react-bootstrap/blob/master/src/InputGroup.js).